x/service/module/autocli.go

```diff
@@ -0,0 +1,52 @@
+package service
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+
+	modulev1 "github.com/pokt-network/poktroll/api/poktroll/service"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		Query: &autocliv1.ServiceCommandDescriptor{
+			Service: modulev1.Query_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Params",
+					Use:       "params",
+					Short:     "Shows the parameters of the module",
```

Comment:
I think this is all ignored for now because we're implementing `AppModule#GetTxCmds()`/`#GetQueryCmds()` but it may be prudent to comment it out with a `TODO(#370): integrate with AutoCLI`.